aiven/service_change.go

```diff
@@ -59,19 +59,24 @@ func backupsReady(service *aiven.Service) bool {
 		return true
 	}
 
+	// no backups for read replicas type of service
+	for _, i := range service.Integrations {
+		if i.IntegrationType == "read_replica" {
```

Comment:
I think this needs to check the direction of the integration. The read_replica integration exists for the source service too. The service is only read-replica if destination service id matches the services own id.

Reply:
thanks, I will change that